src/StringFirstLevel.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Objects;
+
+public class StringFirstLevel {
+
+    public String helloName(String name) {
+        Objects.requireNonNull(name, "string must not be null");
```

Comment:
Please remove all `Objects.requireNonNull()` from everywhere in this homework. It's not required here as it might affect the result on the remote resource (https://codingbat.com/java), e.g. tests, that are predefined there.